_includes/selector.html

```diff
@@ -638,6 +684,8 @@
                 }
                 if (this.active_additional_packages.includes("TensorFlow") && cuda_version !== "11.2") isDisabled = true;
                 if (this.active_os_ver === 'Ubuntu 22.04' && cuda_version !== '11.8' && this.active_method === 'Docker') isDisabled = true;
+                if (this.active_method === 'Pip' && cuda_version !== '11.8') isDisabled = true;
```

Comment:
What's the restriction here? It should be possible to install pip wheels for users with older CUDA versions installed locally (within the same minor version) assuming that's what this is for.

Reply:
Came from [this line](https://github.com/rapidsai/docs/pull/354/files#diff-b1b679a811f38c824cce215c29f17a75472d763031023c125d190e579f723b34R267) in the install docs page stating pip users should be on 11.8 or above.

If that's no longer the case, I imagine we should change it there too, right?

Reply:
That section tells users to refer to the "Troubleshooting and Known Issues" section, but then there is no corresponding information there. The Google Doc where we originally collected these limitations says that the requirement is
> CUDA >= 11.2, with at least the v460.27.03 driver

That seems more likely to be correct to me. We basically just need something with CEC support at runtime.

Reply:
Just want to confirm - I should change the text to:
> CUDA >= 11.2, with at least the v460.27.03 driver

and allow for any cuda 11 in the selector, right?

Reply:
Yep I think so. Anything listed in the initial "provision" section is also valid here AFAIK.

Reply:
Updated in the notes and in the selector.

As the CUDA version selected has no impact on the selector, I've removed it when `pip` is selected, but it might make it resize too much and disabled would be better. Thoughts @exactlyallan ?